### PR TITLE
fix: skip check on delete and go ahead on force

### PR DESCRIPTION
**Pull request by ammarkarachi**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli/src/utils/headless-input-utils.ts [hunk 1]
```diff
@@ -29,4 +29,4 @@ export const readHeadlessPayload = async (): Promise<string> => {
   });
 };
 
-export const isYesFlagSet = (context: Context): boolean => context?.exeInfo?.inputParams?.yes;
+export const isYesFlagSet = (context: Context): boolean => context?.exeInfo?.inputParams?.yes || context?.exeInfo?.inputParams?.force;
```
Comment (kaustavghosh06) [inline packages/amplify-cli/src/utils/headless-input-utils.ts:32]: Do we need to set this or the change below is sufficient? Tehnically force flag is not the same as yes right?

Comment (ammarkarachi) [reply packages/amplify-cli/src/utils/headless-input-utils.ts:32]: Yeah you are right `--force` is only used for delete

File: packages/amplify-cli/src/utils/team-provider-migrate.ts [hunk 2]
```diff
@@ -43,7 +43,11 @@ function isInvalidEnvOrPulling(context: Context): boolean {
     return true;
   }
 
-  const isPulling = context.input.command === 'pull' || context.input.command === 'init' || context.input.command === 'env';
+  const isPulling =
+    context.input.command === 'pull' ||
+    context.input.command === 'init' ||
+    context.input.command === 'env' ||
+    context.input.command === 'delete';
```
Comment (jhockett) [inline packages/amplify-cli/src/utils/team-provider-migrate.ts:50]: nit: simplifies to

```ts
  const pullCommands = ['pull', 'init', 'env', 'delete'];
  return pullCommands.includes(context.input.command);
```

**Review by kaustavghosh06 — APPROVED** [review]
LGTM.

**Review by jhockett — APPROVED** [review]

**Pull request merged**